app/pages/AboutPage/AboutPage.module.scss

```diff
@@ -62,7 +62,15 @@
   justify-content: flex-start;
   li {
     flex-shrink: 0;
+    display: flex;
+    align-items: center;
     margin: 20px;
+    max-height: 84px;
+  }
+
+  img {
+    max-height: 84px;
+    max-width: 100%;
   }
```

Comment:
Our original CSS for this was a bit weird, but I think the following would be a cleaner way to enforce the maximum image height and the vertical centering of items without having to introduce a second layer of flexbox:

```suggestion
  align-items: center;

  li {
    flex-shrink: 0;
    margin: 20px;
  }

  img {
    max-height: 84px;
  }
```

To explain this from top to bottom:

- The new `align-items: center` on the `.partnersList` (the `<ul>`) overrides the default value of `stretch`, which would otherwise cause the flex children's heights to always stretch to the height of the largest flex child in that row. Setting it to `center` causes the flex children to have a height that matches their own content's height and to center all the items in a row.
- I reverted the changes you made to the `li` styles, since with the `align-items: center` on the `ul`, these aren't necessary anymore, and so we can get rid of the second layer of flexbox.
- For the image, we still need to set the `max-height`. I _think_ the `max-width` is unnecessary, since it won't really be able to do anything in a horizontally flowing flexbox.

I had to play around with this locally to get this all working, so what I have here may not be optimal, but I think it's got fewer moving pieces to achieve the layout we want. What makes this all tricky is that have to we keep in mind what is determining each element's height and width, and due to flexbox, sometimes it's the parent that constrains the child, but other times it's the child that influences the parent, and it's different for the horizontal and vertical directions. Making things more complicated is that images also have their own, unique resizing rules.

Anyway, it's up to you if you want to take these changes or just go with what you have. We may want to do some larger cleanup of the CSS on this page later on, anyway.